aspnetcore/publishing/iis.md

```diff
@@ -119,45 +123,51 @@ To include the *publish-iis* tool in your application, add entries to the *tools
-
-## Configure the website in IIS
-
-1. In **IIS Manager**, create a new website. Provide a **Site name** and set the **Physical path** to the application's deployment folder that you created. Provide the **Binding** configuration and create the website.
+3. In **IIS Manager**, create a new website. Provide a **Site name** and set the **Physical path** to the application's deployment folder that you created. Provide the **Binding** configuration and create the website.
 
-2. Set the application pool to **No Managed Code**. ASP.NET Core runs in a separate process and manages the runtime.
+4. Set the application pool to **No Managed Code**. ASP.NET Core runs in a separate process and manages the runtime.
 
-> [!NOTE]
-> If you change the default identity of the application pool from **ApplicationPoolIdentity**, verify the new identity has the required permissions to access the application's folder and database.
-
-Open the **Add Website** window.
+5. Open the **Add Website** window.
 
    ![Click Add Website from the Sites contextual menu.](iis/_static/add-website-context-menu-ws2016.png)
 
-Configure the website.
+6. Configure the website.
 
    ![Supply the Site name, physical path, and Host name in the Add Website step.](iis/_static/add-website-ws2016.png)
 
-In the **Application Pools** panel, open the **Edit Application Pool** window by right-clicking on the website's application pool and selecting **Basic Settings...** from the popup menu.
+7. In the **Application Pools** panel, open the **Edit Application Pool** window by right-clicking on the website's application pool and selecting **Basic Settings...** from the popup menu.
 
    ![Select Basic Settings from the contextual menu of the Application Pool.](iis/_static/apppools-basic-settings-ws2016.png)
 
-Set the **.NET CLR version** to **No Managed Code**.
+8. Set the **.NET CLR version** to **No Managed Code**.
 
    ![Set No Managed Code for the .NET CLR Version.](iis/_static/edit-apppool-ws2016.png)
+   
+> [!NOTE]
+> If you change the default identity of the application pool from **ApplicationPoolIdentity**, verify the new identity has the required permissions to access the application's folder and database.
 
-Browse the website.
+## Deploy the application
+Deploy the application to the folder you created on the target IIS server. Web Deploy is the recommended mechanism for deployment. Alternatives to Web Deploy are listed below.
 
-   ![The Microsoft Edge browser has loaded the IIS startup page.](iis/_static/browsewebsite.png)
+### Web Deploy with Visual Studio
+Create a publish profile in Visual Studio and click the **Publish** button to deploy your application. If your hosting provider has support for creating or supplying a publish profile, download their publish profile and import it using the Visual Studio Publish Web dialog.
```

Comment:
All I had from the one I'm merging into this doc is the Publish dialog.

How about this for a link? https://msdn.microsoft.com/en-us/library/dd465337(v=vs.110).aspx

... but it's marked ".NET Framework 4.5"

Side Note: This is very interesting. I'm sure a lot is not updated for our new Core world and with upcoming VS2017, but interesting ... https://www.asp.net/whitepapers/aspnet-web-deployment-content-map

or if you want to go up one level to the main doc ...

https://msdn.microsoft.com/en-us/library/dd394698(v=vs.110).aspx

> WD without VS

You mean along the lines of this? ... https://technet.microsoft.com/en-us/library/dd568996(WS.10).aspx